### PR TITLE
Use solidity/javascript highlighting in various EIPs

### PR DESCRIPTION
**Pull request by fulldecent**
Simple fix

### CODE REVIEW

**Comment by axic** [comment]
@fulldecent how about creating one PR to fix highlighting and we can just merge that?

**Comment by fulldecent** [comment]
I will do that next. Wanted to make sure this got through before I started on that.

**Comment by axic** [comment]
Do you want to use this PR for that? I'll merge #2373.

**Comment by fulldecent** [comment]
@axic You got it. This PR is now comprehensive. Ready for review.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 137 is in state Final, not Draft or Last Call
 - Error checking file EIPS/eip-1485.md
 - EIP 152 is in state Final, not Draft or Last Call
 - EIP 706 is in state Final, not Draft or Last Call
 - EIP 779 is in state Final, not Draft or Last Call
 - File README.md is not an EIP
 - EIP 1062 requires approval from one of (phyrex@portal.network)
 - EIP 1077 requires approval from one of (@3esmit, avsa@ethereum.org)
 - EIP 1132 requires approval from one of (nitika@govblocks.io)
 - EIP 1386 requires approval from one of (a@colourful.land, @james-sangalli)
 - EIP 1387 requires approval from one of (a@colourful.land, @james-sangalli)
 - EIP 1388 requires approval from one of (a@colourful.land, @james-sangalli)
 - EIP 1450 requires approval from one of (howard@startengine.com, david@startengine.com, @johnshiple)
 - EIP 1491 requires approval from one of (@cohabo)
 - EIP 1504 requires approval from one of (rujiahe@gmail.com, @mayun89, cr025@bucknell.edu, liuxuanzhe@pku.edu.cn, @swordghost)
 - EIP 1592 requires approval from one of (cyril.lapinte@mtpelerin.com, laurent.aapro@mtpelerin.com)
 - EIP 1616 requires approval from one of (@fiiiu, @spalladino, @thegostep, @0age, @elopio)
 - EIP 1753 requires approval from one of (@civickai, katrina@civicledger.com, caroline.marshall888@gmail.com, annaelizabethcrowley@gmail.com, @bitcoinbrisbane)
 - EIP 1844 requires approval from one of (@arachnid)
 - EIP 1973 requires approval from one of (@lerajk, @qinjian)
 - EIP 2045 requires approval from one of (@cdetrio, @axic)
 - EIP 2157 requires approval from one of (@ctzurcanu, @loredanacirstea)
 - EIP 2255 requires approval from one of (@danfinlay, @rekmarks)
 - EIP 2266 requires approval from one of (@sebastianelvis, jiangshan.yu@monash.edu, chris.haoyul@gmail.com)
 - EIP 2304 requires approval from one of (nick@ens.domains)
 - EIP 233 requires approval from one of (@axic)
 - EIP 823 requires approval from one of (@kashishkhullar)
 - EIP 875 requires approval from one of (a@colourful.land, @james-sangalli)
 - EIP 926 requires approval from one of (nick@ethereum.org)
 - EIP 927 requires approval from one of (nick@ethereum.org)
 - EIP 999 requires approval from one of (@5chdn)

**Comment by bitcoinbrisbane** [comment]
> Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):
> 
> * EIP 137 is in state Final, not Draft or Last Call
> * Error checking file EIPS/eip-1485.md
> * EIP 152 is in state Final, not Draft or Last Call
> * EIP 706 is in state Final, not Draft or Last Call
> * EIP 779 is in state Final, not Draft or Last Call
> * File README.md is not an EIP
> * EIP 1062 requires approval from one of ([phyrex@portal.network](mailto:phyrex@portal.network))
> * EIP 1077 requires approval from one of (@3esmit, [avsa@ethereum.org](mailto:avsa@ethereum.org))
> * EIP 1132 requires approval from one of ([nitika@govblocks.io](mailto:nitika@govblocks.io))
> * EIP 1386 requires approval from one of ([a@colourful.land](mailto:a@colourful.land), @James-Sangalli)
> * EIP 1387 requires approval from one of ([a@colourful.land](mailto:a@colourful.land), @James-Sangalli)
> * EIP 1388 requires approval from one of ([a@colourful.land](mailto:a@colourful.land), @James-Sangalli)
> * EIP 1450 requires approval from one of ([howard@startengine.com](mailto:howard@startengine.com), [david@startengine.com](mailto:david@startengine.com), @johnshiple)
> * EIP 1491 requires approval from one of (@cohabo)
> * EIP 1504 requires approval from one of ([rujiahe@gmail.com](mailto:rujiahe@gmail.com), @mayun89, [cr025@bucknell.edu](mailto:cr025@bucknell.edu), [liuxuanzhe@pku.edu.cn](mailto:liuxuanzhe@pku.edu.cn), @swordghost)
> * EIP 1592 requires approval from one of ([cyril.lapinte@mtpelerin.com](mailto:cyril.lapinte@mtpelerin.com), [laurent.aapro@mtpelerin.com](mailto:laurent.aapro@mtpelerin.com))
> * EIP 1616 requires approval from one of (@fiiiu, @spalladino, @thegostep, @0age, @elopio)
> * EIP 1753 requires approval from one of (@CivicKai, [katrina@civicledger.com](mailto:katrina@civicledger.com), [caroline.marshall888@gmail.com](mailto:caroline.marshall888@gmail.com), [annaelizabethcrowley@gmail.com](mailto:annaelizabethcrowley@gmail.com), @bitcoinbrisbane)
> * EIP 1844 requires approval from one of (@Arachnid)
> * EIP 1973 requires approval from one of (@lerajk, @qinjian)
> * EIP 2045 requires approval from one of (@cdetrio, @axic)
> * EIP 2157 requires approval from one of (@ctzurcanu, @loredanacirstea)
> * EIP 2255 requires approval from one of (@danfinlay, @rekmarks)
> * EIP 2266 requires approval from one of (@SebastianElvis, [jiangshan.yu@monash.edu](mailto:jiangshan.yu@monash.edu), [chris.haoyul@gmail.com](mailto:chris.haoyul@gmail.com))
> * EIP 2304 requires approval from one of ([nick@ens.domains](mailto:nick@ens.domains))
> * EIP 233 requires approval from one of (@axic)
> * EIP 823 requires approval from one of (@kashishkhullar)
> * EIP 875 requires approval from one of ([a@colourful.land](mailto:a@colourful.land), @James-Sangalli)
> * EIP 926 requires approval from one of ([nick@ethereum.org](mailto:nick@ethereum.org))
> * EIP 927 requires approval from one of ([nick@ethereum.org](mailto:nick@ethereum.org))
> * EIP 999 requires approval from one of (@5chdn)

How do we approve?

File: EIPS/eip-1753.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Smart contracts can be used to embed regulatory requirements with respect to the
 ## Interface 
 
 ### Solidity Example
-```
+```solidity
```
Comment (bitcoinbrisbane) [inline EIPS/eip-1753.md]: Approve

File: EIPS/eip-233.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ The Meta EIP representing the hard fork should move in to the `Accepted` state o
 
 A template for the [Istanbul Hardfork Meta 1679](https://eips.ethereum.org/EIPS/eip-1679) is included below ([source file on Github](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1679.md)):
 
-```
+```markdown
 {% raw %}
```
Comment (axic) [inline EIPS/eip-233.md]: Not sure about this, given it has this "raw" modifier?

Comment (fulldecent) [reply EIPS/eip-233.md]: Yes. Linguist has improved to support Front Matters in ` ` ` markdown.

But thanks for pointing out -- cleaned up at dce9af347852e434c19d75607f27d75fa850401e

Comment (axic) [reply EIPS/eip-233.md]: Can we handle this file in a separate PR? I'm not sure what is the good solution for it, but don't want to delay this PR.

**Comment by fulldecent** [comment]
@axic Done. It's updated at 59bc9716d78abdd86a5fd8634819de57662c763a

**Pull request merged**